main.go

```diff
@@ -506,34 +512,34 @@ func (e *Exporter) Collect(ch chan<- prometheus.Metric) {
 	c, err := memcache.New(e.address)
 	if err != nil {
 		ch <- prometheus.MustNewConstMetric(e.up, prometheus.GaugeValue, 0)
-		log.Errorf("Failed to connect to memcached: %s", err)
+		level.Error(e.logger).Log("Failed to connect to memcached: %s", err)
```

Comment:
It's a key-value system so this should be `"msg", "Failed to connect to memcached", "err", err`

Reply:
done